spec/features/create_item_spec.rb

```diff
@@ -32,15 +32,13 @@
       login_as user
     end
 
-    scenario do
+    xscenario do
```

Comment:
What's goin on here re "xscenario"

Reply:
@crowesn : I was following Notch 8 example for a basic feature test using Allinson Flex.  https://github.com/notch8/essi/blob/master/spec/features/create_image_spec.rb

This baffled me, too, but their test were passing and mine weren't.  It might be part of debug scramble, but they kept it in, too.